lib/OctokitAction.ts

```diff
@@ -62,10 +62,21 @@ export abstract class OctokitAction extends Action {
     }
   }
 
-  protected async moveIssue(issueId: string, transitionName: string): Promise<void> {
-    const transition = await this.jira.findTransition(issueId, transitionName);
-    if (transition != null) {
-      await this.jira.transitionIssue(issueId, transition);
+  protected updatePullRequestTitle(prNumber: number, newTitle: string): Promise<void> {
+    this.log(`Updating PR #${prNumber} title to: ${newTitle}`);
+    return this.updatePullRequest(prNumber, { title: newTitle });
```

Comment:
Is this intentionally non-async method without await? It should work AFAICT, I just want to make sure that it was intentional.

Reply:
Same below

Reply:
It was intentional.
We don't need to do anything with the Promise in `updatePullRequestTitle` and `updatePullRequestDescription` and can pass it to the caller.